charts/ccsm-helm/test/golden/goldenfiles.go

```diff
@@ -27,12 +29,14 @@ func (s *TemplateGoldenTest) TestContainerGoldenTestDefaults() {
 		SetValues:      s.SetValues,
 	}
 	output := helm.RenderTemplate(s.T(), options, s.ChartPath, s.Release, s.Templates)
-	actual := []byte(output)
+	regex := regexp.MustCompile(".+(helm.sh/chart: ).*\n")
```

Comment:
Tbh I tried it before but failed somehow, but had also not enough time and power to concentrate :sweat_smile: :see_no_evil: I think it will be better and will try yours again later :)